### PR TITLE
set navitia_wrapper cache for web service

### PR DESCRIPTION
**Pull request by xlqian**
refactor and use cache for web service

### CODE REVIEW

**Comment by xlqian** [comment]
Jenkins, test this please!

**Comment by xlqian** [comment]
Jenkins, test this please!

**Comment by xlqian** [comment]
Jenkins, test this please!

**Comment by xlqian** [comment]
@pbougue  done: https://github.com/CanalTP/navitia_python_wrapper/pull/7

File: kirin/gtfs_rt/tasks.py [hunk 1]
```diff
@@ -98,13 +98,12 @@ def gtfs_poller(self, config):
...
-        pubdate_cache_timeout = app.config.get('NAVITIA_PUBDATE_CACHE_TIMEOUT', 600)
-        query_cache_timeout = app.config.get('NAVITIA_QUERY_CACHE_TIMEOUT', 600)
-        nav.set_cache(redis, query_cache_timeout, pubdate_cache_timeout)
-
+        nav = navitia_wrapper.Navitia(url=config['navitia_url'],
+                                      token=config['token'],
+                                      timeout=5,
+                                      cache=redis,
+                                      query_timeout=app.config.get('NAVITIA_QUERY_CACHE_TIMEOUT', 600),
+                                      pubdate_timeout=app.config.get('NAVITIA_PUBDATE_CACHE_TIMEOUT', 600)).instance(config['coverage'])
```
Comment (pbougue) [inline kirin/gtfs_rt/tasks.py]: minor: line length (and maybe code should be similar to gtfsrt)

**Pull request merged**